tests/unit/scripting.tcl

```diff
@@ -934,3 +934,70 @@ start_server {tags {"scripting external:skip"}} {
     r eval {return 'hello'} 0
     r eval {return 'hello'} 0
 }
+
+start_server {tags {"scripting resp3 tests"}} {
+    r debug set-disable-deny-scripts 1
+
+    test {test resp3 big number protocol parsing} {
+        r eval {
+            redis.setresp(3);
```

Comment:
maybe we wanna repeat some of these tests in resp2 mode, so that we have coverage of that translation?
same goes for the module RESP tests..

Reply:
Fixed

Reply:
i see you added a loop for the Lua tests, can we easily add something for the module tests?
e.g. it'll test that attributes are skipped for RESP2

Reply:
@oranagra rethinking about it, I am not sure what we are testing here. We say that if resp2 is used then the reply does not even contain resp3 (map, set, double, ...). So do we want to test resp2 parsing (arrays, strings, numbers, ...), I believe this is pretty cover already no?

Reply:
Summarizing:
We decided to added resp2 and resp3 tests. Was added here: https://github.com/redis/redis/pull/9202/commits/c4fa95984b769f5b694924557891ffb247cf0907